CONFIG.md

```diff
@@ -622,11 +622,13 @@ require'lspconfig'.clangd.setup{}
   - ClangdSwitchSourceHeader: Switch between source/header
   
   Default Values:
-    capabilities = default capabilities, with offsetEncoding utf-8
     cmd = { "clangd", "--background-index" }
     filetypes = { "c", "cpp", "objc", "objcpp" }
-    on_init = function to handle changing offsetEncoding
     root_dir = root_pattern("compile_commands.json", "compile_flags.txt", ".git") or dirname
+    on_init = function to handle changing offsetEncoding
```

Comment:
Just a reminder not to edit CONFIG.md, because it's autogenerated so it just adds a commit that will be wiped out.

Reply:
I was wondering about that...thanks.